spec/unit/provider/directory_spec.rb

```diff
@@ -212,7 +212,6 @@
       it "os x 10.10 can write to sip locations" do
         allow(node).to receive(:[]).with("platform_version").and_return("10.10")
         allow(Dir).to receive(:mkdir).and_return([true], [])
-        allow(::File).to receive(:directory?).and_return(true)
```

Comment:
that looks incorrect since it seems to be removing the stub entirely so it'll hit the real filesystem.

i'd expect to see a File.exists? stub change to a File.directory? stub somewhere.